bothub/utils.py

```diff
@@ -324,3 +351,54 @@ class CountSubquery(Subquery):
 
     def __init__(self, queryset, output_field=None, **extra):
         super().__init__(queryset, output_field, **extra)
+
+
+class AbstractService:
+    def get_org_object_pk(self, pk: int):
```

Comment:
Couldn't that be a function outside the class? If you need to use this functionality elsewhere in the code, you will have to rewrite it, causing duplicate code